### PR TITLE
Prettier svg code

### PR DESCRIPTION
**Pull request by patrislav1**
This PR attempts to produce less verbose / more concise SVG code. The rendered graphics should stay the same.

BTW, I tried to make the background rects inherit `height` and `y` attributes from a parent `<g>` element, to reduce repetition, but apparently `<g>` can only impose a style to child elements, not coordinates or dimensions.

### CODE REVIEW

File: cli/testdata/b-it-ul-st.svg [hunk 1]
```diff
@@ -13,16 +13,26 @@
         .bg {
             stroke-width: "0.5px";
         }
+        .bold {
```
Comment (wader) [inline cli/testdata/b-it-ul-st.svg:16]: Nice with classes like this if one wants to do custom styling 👍

**Comment by wader** [comment]
Looks great! tried it out with chrome on macOS and can't see any issues.

But i did notice a thing unrelated to this change (happens without this change also) that powerline.svg is rendered with wrong character dimensions until the windows is reiszed etc then it snaps to correct size... feels like browser bug?

**Comment by wader** [comment]
> BTW, I tried to make the background rects inherit height and y attributes from a parent <g> element, to reduce repetition, but apparently <g> can only impose a style to child elements, not coordinates or dimensions.

Good to know!

**Review by wader — APPROVED** [review]

**Comment by wader** [comment]
First
Screenshot 2024-01-26 at 17 14 46

After resize
Screenshot 2024-01-26 at 17 14 55

Weird and not sure i noticed it before. Anyway, not related to this PR

**Comment by patrislav1** [comment]
> First Screenshot 2024-01-26 at 17 14 46
> 
> After resize Screenshot 2024-01-26 at 17 14 55
> 
> Weird and not sure i noticed it before. Anyway, not related to this PR

I also see it in both Chrome and Firefox; can't imagine this a browser bug in completely different browser engines :thinking:

**Comment by wader** [comment]
> I also see it in both Chrome and Firefox; can't imagine this a browser bug in completely different browser engines 🤔

Huh yes that seems unlikely

**Pull request merged**